src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
I don't find this that much of an issue. Sure, you get stuff mixed a little but that's why these are representations and not resources. Hypermedia is part of the representation, right?